Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml

```diff
@@ -69,13 +69,22 @@
     <string name="vaccination_card_registration_body">"Fügen Sie Ihre Impfzertifikate in die App hinzu, um sie immer bei sich zu haben. Scannen Sie dafür den QR-Code Ihres Dokuments."</string>
     <!-- XBUT: button for Vaccination Certificate Registration Home Card -->
     <string name="vaccination_card_register">"Hinzufügen"</string>
-
     <!-- XHED: Homescreen vaccination status card title -->
     <string name="vaccination_card_status_title">Digitaler Impfnachweis</string>
     <!-- XHED: Homescreen vaccination status card vaccination name -->
     <string name="vaccination_card_status_vaccination_name">SARS-CoV-2 Impfschutz</string>
-    <!-- XTXT: Homescreen vaccination status card vaccination status label -->
+    <!-- XTXT: Homescreen card incomplete vaccination status label -->
     <string name="vaccination_card_status_vaccination_incomplete">Unvollständiger Impfschutz</string>
+    <!-- XTXT: Homescreen card complete vaccination status label -->
+    <plurals name="vaccination_card_status_vaccination_complete" tools:ignore="UnusedQuantity">
+        <item quantity="one">Vollständiger Impfschutz in %1$d Tag</item>
```

Comment:
Can we change to "in einem Tag" instead of "in 1 Tag" -> UA

Reply:
Should be aligned with https://github.com/corona-warn-app/cwa-app-android/pull/3185/files#diff-fcc1b466984b0efa9d6f2acb9a2d2f9a16b4251afd27e101a23b7972fd6c3728R48 also

Reply:
FYI: In the risk card, we also show the digit for eins => "Begegnungen mit niedrigem Risiko an %1$d Tag""

Reply:
@chiljamgossow you are right that would be correct german, but I assume it was decided like this on purpose like it i on the risk card as @LukasLechnerDev mentioned. The number should stand out so thats easier to "scan". But let's wait for our UA colleagues :)

Reply:
Thanks for the feedback, @chiljamgossow , but we'll go with the digit for eins :)